spec/controller/fee_calculator_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FeeCalculatorController, type: :controller do
+  describe 'GET index' do
+    it 'has a 200 status code' do
+      get :index
```

Comment:
In recent versions of RSpec, controller specs are discouraged in favour of request specs. With older applications there are still a lot of controller specs as it takes time to migrate them but for a new project it would be good to use request specs from the start. Fortunately, they are quite similar;

1) Put them inside `spec/requests` without `controller` in the name, so this file can move to `spec/requests/fee_calculator_spec.rb`.
2) Change the type to `:request`.
3) Use a descriptive title instead of the class name.
4) Use paths (eg, '/fee_calculator' or '/') instead of controller method names (eg, `:index`). There are helpers for this that can be discovered with `rails routes`. We can talk about this, if you want.

So this test could become;

```suggestion
RSpec.describe 'Fee Calculator', type: :request do
  describe 'view the calculator' do
    it 'has a 200 status code' do
      get index_path
```